packages/app/pages/api/auth/[...nextauth].ts

```diff
@@ -0,0 +1,67 @@
+import EarthdataLoginProvider from 'auth/providers/earthdata-login'
+import log from 'lib/log'
+import NextAuth, { AuthOptions } from 'next-auth'
+import { getUsersDb } from 'auth/db'
+
+export const authOptions: AuthOptions = {
+    // use our mEditor logger for NextAuth log messages
+    debug: log.level === 'debug',
+    logger: {
+        error(code, metadata) {
+            log.error(code, metadata)
+        },
+        warn(code) {
+            log.warn(code)
+        },
+        debug(code, metadata) {
+            log.debug(code, metadata)
+        },
+    },
+
+    // use JWT tokens for tracking the user's session
+    session: {
+        strategy: 'jwt',
+    },
+
+    // custom page overrides
+    pages: {
+        signIn: '/meditor/signin',
+    },
+
+    // configure one or more authentication providers
+    providers: [
+        // Earthdata Login Provider
+        ...(process.env.AUTH_CLIENT_ID && process.env.AUTH_CLIENT_SECRET
+            ? [
+                  EarthdataLoginProvider({
+                      clientId: process.env.AUTH_CLIENT_ID,
+                      clientSecret: process.env.AUTH_CLIENT_SECRET,
+                  }),
+              ]
+            : []),
+    ],
+
+    callbacks: {
+        async signIn(session) {
```

Comment:
Since this PR makes mEditor stateless (no DB session), we still need a way to create a user account in the "users-urs" collection.

The notifier uses this collection to pull email addresses when sending emails.

This would be unnecessary if we could get those emails from EDL, but alas that API doesn't exist from what I can see